src/oidcop/oauth2/introspection.py

```diff
@@ -44,7 +44,7 @@ def _introspect(self, token, client_id, grant):
             "active": True,
             "scope": " ".join(scope),
             "client_id": client_id,
-            "token_type": token.type,
+            "token_class": token.token_class,
```

Comment:
`token_type` is needed here. Maybe we can release both?

Reply:
Sure! I'll change.